SentryTestUtils/ClearTestState.swift

```diff
@@ -67,5 +67,8 @@ class TestCleanup: NSObject {
         #endif // os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
 
         sentrycrash_scopesync_reset()
+
+        SentrySdkInfo.resetPackageManager()
+        SentrySdkInfo.clearExtraPackages()
```

Comment:
`m`: Do we really need this here? Although I added this some time ago, I'm not a fan of this global reset, and I would prefer that we use it sparingly. I would prefer if we could tear down the global state of the SDKInfo in the tests using it.

Reply:
Since we have this method now I think it should, if possible, correctly reset the state, otherwise it's very confusing.

Reply:
If it's already broken, than I agree it doesn't make sense to add to it. But if till now it was correct, we should add there the packages reset.